{"name": "test_search_ID_Pet", "status": "broken", "statusDetails": {"message": "RuntimeError: No active exception to reraise", "trace": "self = <tests.test_PetDemoProject_api.Test_Pets object at 0x0000022306531FD0>\n\n    @allure.description(\"Поиск животного по ID\")\n    def test_search_ID_Pet(self):\n>       Pet_Demo_api.get_Pet_for_ID()\n\ntests\\test_PetDemoProject_api.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\n    @staticmethod\n    def get_Pet_for_ID():\n        if Pet_Demo_api.ID is None:\n            print(\"ID не установлен\")\n>           raise  # указываем об ошибке\nE           RuntimeError: No active exception to reraise\n\nutils\\API_Pets.py:77: RuntimeError"}, "description": "Поиск животного по ID", "attachments": [{"name": "stdout", "source": "67a84c6d-32c4-4320-9c30-51d8e9088bfe-attachment.txt", "type": "text/plain"}], "start": 1696768305126, "stop": 1696768305127, "uuid": "0feed09f-baff-4292-9474-66863a47c0df", "historyId": "5e2ec9bbcd28ebed347b3460434e14cd", "testCaseId": "e1ca68db3a9dd992253f523298ba507e", "fullName": "tests.test_PetDemoProject_api.Test_Pets#test_search_ID_Pet", "labels": [{"name": "epic", "value": "Управление животными"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_PetDemoProject_api"}, {"name": "subSuite", "value": "Test_Pets"}, {"name": "host", "value": "LAPTOP-DQ0BFIQ6"}, {"name": "thread", "value": "12036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_PetDemoProject_api"}]}